packager/app/test/packager_test.py

```diff
@@ -575,6 +575,9 @@ def _GetFlags(self,
     elif force_cl_index is False:
       flags += ['--noforce_cl_index']
 
+    if start_segment_number:
```

Comment:
Maybe `if start_segment_number is not None:` so that a start number of 0 is permitted.